### ISSUE TITLE
aws_security_group - description doesn't change

### ISSUE THREAD

**Issue by blacked** [hashicorp/terraform#1267]
```
# saltmaster
resource "aws_security_group" "satmaster" {
    name = "saltmaster"
    description = "${concat(var.environment, \" saltmaster111\")}"
    ...
}
```

```
aws_security_group.satmaster: Modifying...
  description: "eoe-cul saltmaster" => "cep-cul saltmaster1111 "
```

But nothing was changed.

**Comment by blacked** [comment]
`Terraform v0.4.0-dev`

**Comment by catsby** [comment]
The AWS API for Security Groups doesn't allow update at this time:

- http://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Operations.html

Unfortunately, `description` here should be a `ForceNew` operation

**Comment by bts** [comment]
This should probably still be considered a "bug" though, right?

If a user wants to change the description on an ELB, they can't do so through terraform -- they have manually delete the resource (and its dependencies) and then rebuild with terraform.

**Comment by jszwedko** [comment]
I agree, this still feels like a bug @catsby

**Comment by inferiorhumanorgans** [comment]
Looks like this was resolved in #1871

**Comment by phinze** [comment]
Yep - fixed in #1871 - after that the description change will be detected and the SG will be replaced to apply the change.
